style.css

```diff
@@ -192,6 +192,14 @@ footer {
   background-color: #ffb6c1;
 }
 
+#warning {
+  background: #d5e47f;
```

Comment:
Not a huge fan of the color, but I'm also not known for my good taste. Maybe something more like a warning-orange? :) It's OK to leave this if you feel strong.y

Reply:
I made it little more vivid in https://github.com/mozilla/srihash.org/pull/518/commits/8698f0dc093d7b0d24042ac421444a6820d87170 . Yellow looks better as a warning than orange in my opinion. Bright yellow is also better for accessibility.